Sprintfinity3902/Entities/Link/Player.cs

```diff
@@ -87,7 +89,7 @@ public Player(Game1 _game)
             _collidable = true;
             SetStepSize(1);
             MaxHealth = INITIAL_HEALTH;
-            LinkHealth = MaxHealth;
+            LinkHealth = 1;//TODO: MAX_HEALTH undid for testing lose state
```

Comment:
This definitely makes testing the lose wrapper easier, but I think for Master it should be changed back to MaxHealth to start!